notification-api.md

```diff
@@ -156,84 +158,82 @@ that should be announced beyond the immediate effect of the primary action.
 
 ## Proposed Solution
 
-(This version inspired by [UIA Notification API](https://docs.microsoft.com/en-us/windows/win32/api/uiautomationcoreapi/nf-uiautomationcoreapi-uiaraisenotificationevent))
-
-For use cases that don't have an express UI tie-in, it makes sense to provide a solution
-not expressly tied to HTML element, but for which other document-centric information can
-be inferred (such as language).
+We provide an API on Documents and Elements to enable posting imperative notifications with 
+independently-managable queues. Additionally, Element-based context should be assumed for
+language.
```

Comment:
`Additionally, Element-based context should be assumed for
language.`

Nit: The wording confused me a bit, maybe clarify as 'The language of the notification is based on the element on which the notification is (invoked/fired?)'

Reply:
Good suggestion.